### DISCUSSION TITLE
Docker compose-command in lxc

### DISCUSSION THREAD

**Discussion by knorst1** [tteck/Proxmox#2396] [Help Needed] [answered]
Hi,
is it possible to use the command docker compose up -d and likewise other docker commands inside the lxc container of paperless ngx (helper scripts version)? I did not succeed so.far
Thank you for your help

**Comment by tteck** [comment, marked as answer]
No, it's not a Docker install.

**Reply by knorst1** [reply]
Ok thank you, 
So for the paperless configuration I have to use 'paperless.conf' to change the config, I suppose..?
